prow/config/jobs_test.go

```diff
@@ -118,9 +114,6 @@ func TestPresubmits(t *testing.T) {
 					if job.Context == job2.Context {
 						t.Errorf("Jobs %s and %s have the same context: %s", job.Name, job2.Name, job.Context)
 					}
-					if job2.re.MatchString(job.RerunCommand) {
-						t.Errorf("%d, %d, RerunCommand \"%s\" from job %s matches \"%v\" from job %s but shouldn't.", i, j, job.RerunCommand, job.Name, job2.Trigger, job2.Name)
-					}
```

Comment:
Unclear why this check is being removed.

Reply:
because it fails incorrectly, when the rerun is unset and therefore defaulting this spuriously fails. I'm not sure why we even had this check.

Reply:
As mentioned above, It is not valid for rerun or trigger to be unspecified at this point. Those fields should be defaulted when the config is loaded in TestMain.

Reply:
yeah the problem is I cannot serialize back from a job and preserve this. I load the jobs, modify them to translate from k/k to k-s/k, then serialize them back. because this field is magically mutated on load and not a pointer, the best I can do is write `""` which is insufficient currently.